test/unittests/cli/test_helper_functions.py

```diff
@@ -34,95 +34,91 @@ def setUpClass(cls) -> None:
                 "https://api.dasch.swiss",
-                "https://iiif.dasch.swiss",
+                "https://ingest.dasch.swiss",
             ],
             "https://api.dasch.swiss": [
                 "https://api.dasch.swiss",
-                "https://iiif.dasch.swiss",
+                "https://ingest.dasch.swiss",
             ],
             "https://app.dasch.swiss": [
                 "https://api.dasch.swiss",
-                "https://iiif.dasch.swiss",
+                "https://ingest.dasch.swiss",
             ],
-            "https://iiif.dasch.swiss": [
+            "https://ingest.dasch.swiss": [
                 "https://api.dasch.swiss",
-                "https://iiif.dasch.swiss",
+                "https://ingest.dasch.swiss",
             ],
             "https://dasch.swiss": [
                 "https://api.dasch.swiss",
-                "https://iiif.dasch.swiss",
+                "https://ingest.dasch.swiss",
             ],
             "dasch.swiss": [
                 "https://api.dasch.swiss",
-                "https://iiif.dasch.swiss",
+                "https://ingest.dasch.swiss",
             ],
             "http://admin.test.dasch.swiss/": [
                 "https://api.test.dasch.swiss",
-                "https://iiif.test.dasch.swiss",
+                "https://ingest.test.dasch.swiss",
             ],
             "http://app.staging.dasch.swiss/": [
                 "https://api.staging.dasch.swiss",
-                "https://iiif.staging.dasch.swiss",
+                "https://ingest.staging.dasch.swiss",
             ],
             "https://demo.dasch.swiss/": [
                 "https://api.demo.dasch.swiss",
-                "https://iiif.demo.dasch.swiss",
+                "https://ingest.demo.dasch.swiss",
             ],
             "http://api.dev.dasch.swiss/": [
                 "https://api.dev.dasch.swiss",
-                "https://iiif.dev.dasch.swiss",
+                "https://ingest.dev.dasch.swiss",
             ],
             "dev-02.dasch.swiss": [
                 "https://api.dev-02.dasch.swiss",
-                "https://iiif.dev-02.dasch.swiss",
+                "https://ingest.dev-02.dasch.swiss",
             ],
             "082a-test-server.dasch.swiss": [
                 "https://api.082a-test-server.dasch.swiss",
-                "https://iiif.082a-test-server.dasch.swiss",
+                "https://ingest.082a-test-server.dasch.swiss",
             ],
             "admin.08F4-test-server.dasch.swiss": [
                 "https://api.08F4-test-server.dasch.swiss",
-                "https://iiif.08F4-test-server.dasch.swiss",
+                "https://ingest.08F4-test-server.dasch.swiss",
             ],
             "app.08F4-test-server.dasch.swiss": [
                 "https://api.08F4-test-server.dasch.swiss",
-                "https://iiif.08F4-test-server.dasch.swiss",
-            ],
-            "iiif.E5bC-test-server.dasch.swiss": [
```

Comment:
Yes, this behaviour is not supported by the current code anymore because iiif was replaced by ingest. 

I have added a test for ingest instead in d9fed83c87701b1e637868ffe1b02a704f2e37c8